### PR TITLE
Add toast for actions

### PR DESCRIPTION
**Pull request by aishwarya-bm**
added toast for user actions
added small responsiveness changes on productlist page for filters
minor changes to show search box only on product list page

### CODE REVIEW

**Review by sadathshariff — COMMENTED** [review]
Looks Good, like how you have made Toast as Component and re-using it

**Review by sadathshariff — APPROVED** [review]

File: src/components/toast/Toast.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Slide, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+export default function Toast({ message, type }) {
+  return toast(message, {
+    position: "bottom-right",
+    autoClose: 5000,
```
Comment (nishasen) [inline src/components/toast/Toast.jsx]: you can give smaller timer as api is not taking that much time, like 1000 or 2000

File: src/pages/productlist/Productlist.jsx [hunk 2]
```diff
@@ -41,7 +41,7 @@ export default function Productlist() {
   }
 
   const isProductInCart = p_id => {
-    return cart.find(p => p._id === p_id);
+    return cart?.find(p => p._id === p_id);
```
Comment (nishasen) [inline src/pages/productlist/Productlist.jsx:44]: I like your this conditioning part

**Review by nishasen — APPROVED** [review]
I code is neat and good.

File: src/components/toast/Toast.jsx (hunk 1 above)
Comment (aishwarya-bm) [reply src/components/toast/Toast.jsx]: ya i had same thought, thanks, I changed it to 3 secs

**Pull request merged**